Addressing imposter syndrome

EYezerets opened the pull request:
I believe that, as part of the lab's discussion of how to address the impacts of systemic racism in the United States in an academic context, it may also be helpful to offer guidance to mentors on how to support their mentees who may be more likely to experience imposter syndrome in addition to other challenges. If this is outside the scope of the lab agreement and would be more valuable as a training exercise, or if it refocuses the conversation in an unhelpful direction, I also understand that decision. Here's just a starting point! Thanks for your time

EYezerets committed:
Addressing imposter syndrome

I believe that, as part of the lab's discussion of how to address the impacts of systemic racism in the United States in an academic context, it may also be helpful to offer guidance to mentors on how to support their mentees who may be more likely to experience imposter syndrome in addition to other challenges. If this is outside the scope of the lab agreement and would be more valuable as a training exercise, or if it refocuses the conversation in an unhelpful direction, I also understand that decision. Here's just a starting point! Thanks for your time

tathey1 commented on content/about/agreements.md:
```diff
@@ -79,6 +79,13 @@ Some links that might be useful to understand best practices for giving and rece
 
 Feedback for jovo can be provided directly in the #jovofeedback slack channel, including anonymous feedback.
 
+Mentors can anticipate that many of their mentees may experience imposter syndrome. This can be compounded by stereotype threat and experiences of discrimination that have happened in the past or their current environment (which should be addressed by the team). Mentors can support their mentees (and vice versa) by educating themselves on imposter syndrome’s causes, indications, and counteractive ...
```
Very good! In the second article there was an idea I liked that encouraged leaders to show their own uncertainties and learning process. If others agree, I think this could be included in your sentence, "...to use positive and constructive feedback to affirm their value, to share their own uncertainties and learning process, and to ask..."

bdpedigo commented:
@EYezerets this looks like a great addition, thanks for making the PR. I'm wondering if this would make sense to either have as its own subsection, or put under the the "Radical Honesty and Safety" subsection? Also, I am trying to think of ways to make our agreements more concrete/actionable. It sounds like what you are proposing is that lab members commit to learning about impostor syndrome and how to deal with it, is that a fair summary?

EYezerets commented:
Thank you for all the feedback! It seems like it would make sense to update the wording as @tathey1 and @bdpedigo suggested to make sure mentors acknowledge their own learning process and commit to reading the resources I listed (in addition to continuing to educate themselves on being a mentor). However, I will make a new PR that will be in the Mentorship section. Does this sound like a reasonable summary of our discussion?

bdpedigo commented:
> ested to make sure mentors acknowledge their own learning process and commit to reading the resources I listed (in addition to continuing to educate themselves on being a mentor). However, I will make a new PR that will be in the Mentorship section. Does this sound like a reasonable summary of our discussion

Hi @EYezerets I believe this is what we discussed yes. But no need to make a new PR, PRs can be updated by adding additional commits. You can either directly edit on github and it will update, or push commits from a local branch to this one. Let me know if that process is unclear and we can talk about it or i can find you some github resources

EYezerets committed:
Update after 6/24 lab meeting

I have moved the suggestion to the Mentorship section and clarified the language. The action item is that team members are requested to read the 3 resources listed. Thanks in advance for your feedback!

EYezerets commented:
@bdpedigo Thanks! I think I figured out how to make the edits. Please kindly let me know if you are able to see the update!

bdpedigo commented:
Hi @EYezerets , yes i see it. I wonder if it makes sense to make these actual commitments (i.e. each mentor will read the following) or not? If not, sounds like should go in aspirational goals (even though these look like great resources). This is not to say that we shouldn't come up with a goal everyone does have to be accountable, but just that we want to be able to say whether or not people are doing the stuff that is under "agreements."

EYezerets commented:
@bdpedigo Good point, I can move it there.

EYezerets committed:
Update - moved to Aspirational Agreements section
